StarkInfra/StarkInfra/CreditNote/Transfer.cs

```diff
@@ -19,10 +19,10 @@ namespace StarkInfra
     ///     <item>BranchCode [string]: receiver Infra account branch. Use '-' in case there is a verifier digit. ex: "1357-9"</item>
```

Comment:
Em cima dessa linha, na linha 18, a docstring do **bankCode** foi afetada por algum replace all de Bank para Infra.

Deveria ser: `code of the receiver bank institution in Brazil. ex: "20018183" or "341"`

Da um check geral na palavra "Infra" só pra conferir se não teve outros "bank" que foram trocados erroneamente.

Reply:
Aqui na linha 19 também está errado, deveria ser `receiver bank`, não `receiver Infra`.